src/scenic/syntax/compiler.py

```diff
@@ -240,6 +240,7 @@ class PropositionTransformer(Transformer):
     def __init__(self, filename="<unknown>") -> None:
         super().__init__(filename)
         self.nextSyntaxId = 0
+        self.in_atomic = False
```

Comment:
```suggestion
        self.inAtomic = False
```
(for consistency with the rest of this file and most of Scenic)